skiko/src/commonMain/kotlin/org/jetbrains/skia/impl/Native.kt

```diff
@@ -44,16 +44,18 @@ expect class InteropScope() {
     fun release()
 }
 
+typealias InteropScopeContext = InteropScope.(InteropPointer) -> Unit
+
 expect inline fun <T> interopScope(block: InteropScope.() -> T): T
 
-inline fun withResult(result: ByteArray, block: (InteropPointer) -> Unit): ByteArray = interopScope {
+inline fun withResult(result: ByteArray, block: InteropScopeContext): ByteArray = interopScope {
     val handle = toInterop(result)
     block(handle)
     handle.fromInterop(result)
     result
 }
 
-inline fun withNullableResult(result: ByteArray, block: (InteropPointer) -> Boolean): ByteArray? = interopScope {
+inline fun withNullableResult(result: ByteArray, block: InteropScope.(InteropPointer) -> Boolean): ByteArray? = interopScope {
```

Comment:
Cool. Maybe also introduce alias for `InteropScope.(InteropPointer) -> Boolean`?

Reply:
We have one or two places with such signature and I decided it would be overkill.
It's a pity we don't have default type params in Kotlin